datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -391,6 +391,12 @@ impl ParquetExecBuilder {
             &projected_output_ordering,
             &base_config,
         );
+
+        base_config
```

Comment:
> that is an interesting idea

But it still seem to make some long queries slower...

After finish the refactor work about blocked accumulators sketch. I plan to try to refactor the expansive clone scalar values like String to Arc<str>, and see if it can faster the shorts and don't slower the longs...